orm/model/ormtable/backend.go

```diff
@@ -172,10 +179,6 @@ type contextKeyType string
 
 var defaultContextKey = contextKeyType("backend")
 
-func getBackendDefault(ctx context.Context) (Backend, error) {
-	return ctx.Value(defaultContextKey).(Backend), nil
-}
-
-func getReadBackendDefault(ctx context.Context) (ReadBackend, error) {
+func getBackendDefault(ctx context.Context) (ReadBackend, error) {
 	return ctx.Value(defaultContextKey).(ReadBackend), nil
```

Comment:
i've had some annoying debug sessions from context issues cause of patterns like this in the SDK, perhaps we should check if the type cast is successful, and return a more expressive error if not?

Reply:
sure

Reply:
updated